components/classes/asset.schema.json

```diff
@@ -0,0 +1,91 @@
+{
+  "meta:license": [
+    "Copyright 2025 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/classes/asset",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Asset",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:extends": ["https://ns.adobe.com/xdm/data/record"],
+  "description": "Digital asset such as an image or video.",
+  "definitions": {
+    "asset": {
+      "properties": {
+        "xdm:asset": {
+          "title": "Asset",
+          "description": "Digital asset such as an image or video.",
+          "type": "object",
+          "properties": {
+            "xdm:version": {
+              "title": "Version",
+              "type": "string",
+              "description": "Asset version."
+            },
+            "xdm:perceptionID": {
+              "title": "Perception ID",
+              "type": "string",
+              "description": "Unique identifier of the asset perception."
+            },
+            "xdm:ID": {
```

Comment:
@anandphatak - Should we have this ID or should we just use the _id inherited from /xdm/data/record

Reply:
You can use inherited _id if the value it will contain is a unique identifier.

Reply:
This should be the unique identifier so we will use the record _id